PurrSoft/PurrSoft.Application/Models/FosterDto.cs

```diff
@@ -0,0 +1,31 @@
+﻿using PurrSoft.Domain.Entities.Enums;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace PurrSoft.Application.Models
+{
+	public class FosterDto
+	{
+		public string UserId { get; set; }
+		public DateTime StartDate { get; set; }
+		public DateTime? EndDate { get; set; }
+		public FosterStatus Status { get; set; }
```

Comment:
if you put an enum directly it forces the frontend to send numbers from enums and confusions can arise really easily.
In my opinion you should get a string and try to convert it to the enum

Reply:
Changed to string and conversion.